service/src/main/resources/db/changesets/20230530_avoid_postgres_specific_features.yaml

```diff
@@ -76,12 +92,12 @@ databaseChangeLog:
                   remarks: Deleting a concept set will cascade to delete the criteria contained in it
               - column:
                   name: key
-                  type: text
+                  type: ${_text.type}
```

Comment:
Same as above. This column is part of the unique constraint, does it need to be `id.type`?

Reply:
yes, if it is part of unique constraint, varchar(255) is OK too if key length if >50.